### PR TITLE
Update 02.manuscript.md with outline

### PR DESCRIPTION
**Pull request by dcsoto**
I added a preliminary manuscript outline as discussed in our previous meeting.

### CODE REVIEW

File: content/02.manuscript.md [hunk 1]
```diff
@@ -1,12 +1,54 @@
...
+Topics:
+- Literate programming: R Markdown, Jupyter Notebooks
+- Version control: Git / GitHub (commits)
+- Software versioning and environment managers (Conda, python-env)
+- Modularize, snippets
+- Coding style: variable naming, linter, pep8, commenting
+- Programming practices: paradigms (object-oriented, procedural), assertions, pair-programming
 
-### Subtitle
+## Level 2: Collaborative
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]: how about: 

```suggestion
## Level 2: Collaboration
```

File: content/02.manuscript.md [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+- Importance to reproducibility in computational biologt.
+- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs, 
+- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
+- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
+- Structure of this manuscript: mention Figure 1 and Figure 2.
+  - **Figure 1:** Funnel structure for better computational biology.
+  - **Figure 2:** Types of computational biology projects per level.
 
-Some text.
+## Level 1: Personal
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]: How about:

```suggestion
## Level 1: Individual Research
```

**Review by BenjaSanchez — COMMENTED** [review]
Overall looks great! Some 1st suggestions below

File: content/02.manuscript.md [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
 
-## Title 1
+- Importance of computation in biological research and scope.
+- Importance to reproducibility in computational biologt.
+- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs, 
+- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
+- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
+- Structure of this manuscript: mention Figure 1 and Figure 2.
+  - **Figure 1:** Funnel structure for better computational biology.
+  - **Figure 2:** Types of computational biology projects per level.
```
Comment (BenjaSanchez) [inline content/02.manuscript.md:10]: I would introduce figure 2 later in the manuscript (In the "All three levels in action" section)

File: content/02.manuscript.md [hunk 4]
```diff
@@ -1,12 +1,54 @@
+## Introduction
 
-## Title 1
+- Importance of computation in biological research and scope.
+- Importance to reproducibility in computational biologt.
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]:
```suggestion
- Importance to reproducibility in computational biology.
```

File: content/02.manuscript.md [hunk 5]
```diff
@@ -1,12 +1,54 @@
...
+Topics:
+- Sharing notebooks: R Markdowns and Jupyter Notebooks (Binder, Google CoLab)
+- Sharing apps: Shiny apps, Dashboard.
+- GitHub (branching, pull requests)
+- Workflow automation: Snakemake (NextFlow, Make, Bash script)
+- Sharing data and metadata
+
+## Level 3: Community
+
+Goal: How to develop and mantain a computational biology project with community feedback over time?
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]:
```suggestion
Goal: How to develop and maintain a computational biology project with community feedback over time?
```

File: content/02.manuscript.md [hunk 6]
```diff
@@ -1,12 +1,54 @@
...
+
+Topics:
+- GitHub releases and semantic versioning
+- Git Flow, GitHub Issues
+- Continous integration and unit tests
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]:
```suggestion
- Continuous integration and unit tests
```

Comment (BenjaSanchez) [inline content/02.manuscript.md]: I feel like unit tests is something useful in a project from the beginning, but for sure already at the "Collaboration" level... maybe move up? CI can stay here.

File: content/02.manuscript.md [hunk 7]
```diff
@@ -1,12 +1,54 @@
...
+- Dependencies per user: pip-tools
+- Sharing software as Python packages, Conda/Bioconda or containers (Docker, Singularity)
+- Include license (MIT) and DOIs
+- Documentation: read the docs.
+
+Note: Maybe add software development methodologies (Agile, Scrum, Sprints).
```
Comment (BenjaSanchez) [inline content/02.manuscript.md:48]: good idea! I can talk a bit about Zenhub here :)

File: content/02.manuscript.md [hunk 8]
```diff
@@ -1,12 +1,54 @@
...
+
+## All three levels in action
+
+- Example of computational biology project 1: RNA-seq analysis (workflow)
+- Example of computational biology project 2: Genome-scale metabolic model (community resource)
```
Comment (BenjaSanchez) [inline content/02.manuscript.md]: not sure if a GEM intended to be a "community resource" fits this example, as then they should be treated more as computational tools... how about instead:

```suggestion
- Example of computational biology project 2: Genome-scale metabolic model (systems biology project)
```

**Review by BenjaSanchez — COMMENTED** [review]
Added a few suggestions and fixed a few spelling errors, after that it will be ready to be merged :)

**Comment by BenjaSanchez** [comment]
@dcsoto let me know here if something in my review was not clear :)

**Comment by dcsoto** [comment]
@BenjaSanchez I think you need to approve some reviews here.

**Comment by BenjaSanchez** [comment]
@dcsoto you clicked on "resolve conversation" for all pending issues, but there were some suggestions there, which means you can go to "Files changed" and then click on "Add suggestion to batch" if you agree with them. I can also just do that myself, let me know :)

**Comment by dcsoto** [comment]
@BenjaSanchez Thanks for the info! I think I'm doing something wrong because it doesn't let me approve the PR. Can you help me with this?

**Review by BenjaSanchez — APPROVED** [review]
@dcsoto Github won't let you approve the PR because you opened it ;) but here it is approved! ready to merge

**Pull request merged**